tests/helper/helper_cluster.go

```diff
@@ -13,36 +13,36 @@ import (
 	"github.com/onsi/gomega/gexec"
 )
 
-type OcRunner struct {
-	// path to oc binary
+type ClusterRunner struct {
+	// path to cluster binary
 	path string
 }
 
-// NewOcRunner initializes new OcRunner
-func NewOcRunner(ocPath string) OcRunner {
-	return OcRunner{
-		path: ocPath,
+// NewClusterRunner initializes new ClusterRunner
+func NewClusterRunner(clusterPath string) ClusterRunner {
```

Comment:
Literally ClusterRunner looks like you are provisioning or running a cluster through the script, isn't it ? I would suggest a better phrase for what exactly you want to do, for example ```func NewClirRunner(clusterPath string) CliRunner {```. I mean replace ```ClusterRunner``` with ```CliRunner```

@prietyc123 Does it make sense ?

Reply:
Done